spec/lib/web_of_science/record_spec.rb

```diff
@@ -224,6 +224,7 @@
     end
   end
 
+=begin
```

Comment:
I don't see any useful changes to this file.  What are you intending w/ these `=begin` and `=end` tags?

Reply:
That was just putting things in comment as per the IDEA